commit ebb0cd55043c4d05af4c5ab3016e96ed5a6bec8c
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Aug 23 15:35:04 2018 +0100

    BCM270X_DT: Sync downstream VCHIQ node w. upstream
    
    Retain a vestigial cache_line_size parameter to improve backwards-
    compatibility.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2708-rpi.dtsi b/arch/arm/boot/dts/bcm2708-rpi.dtsi
index fe2694a1cbdc..00160e0b8ab7 100644
--- a/arch/arm/boot/dts/bcm2708-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi.dtsi
@@ -35,7 +35,6 @@
 		usb = &usb;
 		leds = &leds;
 		fb = &fb;
-		vchiq = &vchiq;
 		thermal = &thermal;
 		axiperf = &axiperf;
 	};
@@ -67,12 +66,10 @@
 			status = "disabled";
 		};
 
-		vchiq: vchiq {
+		mailbox@7e00b840 {
 			compatible = "brcm,bcm2835-vchiq";
-			reg = <0x7e00b840 0xf>;
+			reg = <0x7e00b840 0x3c>;
 			interrupts = <0 2>;
-			cache-line-size = <32>;
-			firmware = <&firmware>;
 		};
 
 		vcsm: vcsm {
@@ -95,7 +92,7 @@
 	};
 
 	__overrides__ {
-		cache_line_size = <&vchiq>, "cache-line-size:0";
+		cache_line_size;
 
 		uart0 = <&uart0>,"status";
 		uart1 = <&uart1>,"status";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1201_ebb0cd55043c4d05af4c5ab3016e96ed5a6bec8c.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1201_ebb0cd55043c4d05af4c5ab3016e96ed5a6bec8c.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
